contracts/voting_escrow_delegation/tests/simulation.rs

```diff
@@ -0,0 +1,399 @@
+use anyhow::Result;
+use astroport_governance::utils::WEEK;
+
+use crate::test_helper::{mock_app, Helper};
+use cosmwasm_std::{Addr, Uint128};
+use cw_multi_test::{next_block, App, AppResponse};
+mod test_helper;
+
+#[derive(Clone, Debug)]
+enum Event {
+    CreateDelegation(Uint128, u64, String, String),
+    ExtendDelegation(Uint128, u64, String),
+}
+
+use Event::*;
+
+struct Simulator {
+    helper: Helper,
+    router: App,
+}
+
+impl Simulator {
+    fn new<T: Clone + Into<String>>(_users: &[T]) -> Self {
```

Comment:
```suggestion
    fn new() -> Self {
```
users array is not used thus no need to pass it as variable